cmd/agree/agree.go

```diff
@@ -192,24 +194,35 @@ var userAnswer = func() (string, error) {
 	return bufio.NewReader(os.Stdin).ReadString('\n')
 }
 
-func parseTermRevision(s string) (string, int, error) {
-	fail := func(err error) (string, int, error) {
-		return "", -1, err
+func parseTermRevision(s string) (string, string, int, error) {
```

Comment:
@cmars I agree with all of this but I will do it in a follow up, at the moment this function exactly the same as the one in terms service, I'd rather just leave it as is for now (no better or worse than the existing then get it right in the charm package)